packages/app/src/app/overmind/namespaces/comments/actions.ts

```diff
@@ -523,22 +499,34 @@ export const onCommentAdded: Action<CommentAddedSubscription> = (
   { state },
   { commentAdded: comment }
 ) => {
-  state.comments.comments[comment.sandbox.id][comment.id] = comment;
+  // We only add it if it was NOT created by the user, cause then we
+  // have already set it, avoiding UI flickers
+  if (comment.user.id !== state.user?.id) {
+    state.comments.comments[comment.sandbox.id][comment.id] = comment;
+  }
 };
 
 export const onCommentChanged: Action<CommentChangedSubscription> = (
   { state },
   { commentChanged: comment }
 ) => {
-  Object.assign(
-    state.comments.comments[comment.sandbox.id][comment.id],
-    comment
-  );
+  // We only change it if it was NOT created by the user, cause then we
+  // have already set it, avoiding UI flickers
+  if (comment.user.id !== state.user?.id) {
```

Comment:
Same here!

Reply:
Jup, should be okay here as well after merge from master (like, removing it)